### ISSUE TITLE
SPLASH SCREEN: Should the "Soundscape" name be in White for easier visibility?

### ISSUE THREAD

**Issue by StevenAbrams** [soundscape-community/soundscape#54]
**Build: 1.0.0 (9)**

**Repro Steps:**
1. Launch the app
2. View the splash screen and the word "Soundscape"

**Expected Results:**
The word "Soundscape" will be clear to see, e.g., in White on the blue background

**Actual Results:**
The "Soundscape" name is in Black and difficult to see against the blue background.

See the image:
 
![image](https://github.com/soundscape-community/soundscape/assets/6051244/45b3394e-c945-4fd6-ab67-b8c7a2935245)

**Comment by steinbro** [comment]
Agreed this isn't ideal, especially for an app with a large low-vision user base. I'm not sure why this color was the default; the Microsoft Soundscape release colored this appropriately.

**Comment by StevenAbrams** [comment]
@steinbro @Oliver2213 

Re-opening. Will verify the fix on the next build. Please confirm which build this fix will appear in.
Thanks

**Comment by Oliver2213** [comment]
@StevenAbrams fixed in build 10.

**Comment by StevenAbrams** [comment]
Verified on 1.0.0 (13). Closing @Oliver2213
